integration/README.md

```diff
@@ -22,3 +22,30 @@
     ```bash
     $ ./bin/dmsg-server ./integration/configs/dmsgserver1.json
     ```
+
+## Take dmsg-server under load
+You need [tmux](https://github.com/tmux/tmux) to continue this test
+1. Start a local dmsg environment
+2. Clone skywire, checkout to develop
+    ```
+    git clone https://github.com/skycoin/skywire.git underLoadDmsgServer
+    cd underLoadDmsgServer
+    git checkout develop
+    ```
+3. Add `time.Sleep(15 * time.Minute)` at the beginning of `initLauncher` at **pkg > visor > init.go**.
+    ```
+    ...
+    func initLauncher(ctx context.Context, v *Visor, log *logging.Logger) error {
+	  time.Sleep(15 * time.Minute)
+	  conf := v.conf.Launcher
+    ...
+    ```
+4. Build new binaries: `make build`
+5. Copy `underLoadLocal.sh` file to skywire clone directory
+6. Run it by `sh underLoadLocal.sh -n 200`
```

Comment:
```
erson@erson-69:~/Workspace/Go/skywire$ sh underLoadLocal.sh -n 200
underLoadLocal.sh: 3: Syntax error: "(" unexpected
```
this gives error
`bash underLoadLocal.sh -n 200` works

Reply:
Mh. It work for me with `sh`. I will change it to bash in README.md.